services/blockchain-indexer/shared/dataService/blocks.js

```diff
@@ -114,6 +114,23 @@ const getBlocks = async (params = {}) => {
 	};
 };
 
+const getBlocksAssets = async params => {
+	const blocksAssets = {
+		data: [],
+		meta: {},
+	};
+
+	if (params.blockID) logger.debug(`Retrieving block assets for the block with ID ${params.blockID} from Lisk Core`);
+	else if (params.height) logger.debug(`Retrieving block assets for the block at height: ${params.height} from Lisk Core`);
+	else logger.debug(`Retrieving block assets with custom search: ${util.inspect(params)} from Lisk Core`);
```

Comment:
Please consider moving these debug logs to the lower layer (`business`) implementation.